### DISCUSSION TITLE
Home? Where is home?

### DISCUSSION THREAD

**Discussion by onjaysantana** [givanz/Vvveb#125] [Q&A]
Hi, I'm still having trouble installing vvVeb. I have an ISP with a typical Linux folder setup. Each domain is a folder within a DOMAINS folder. The public site is in a sub-folder of the domain, called PUBLIC_HTML.  

Questions:

-- Where do I setup vvVeb? What I have tried gets the program to open, but all the pages "don't exist" even though they are listed in vvVeb. That includes the demo folder. 

-- I would like to set this up so that I can manage the CMS for multiple sites (domains) on the ISP. Again, the folder layout is unclear to me. The response I got when I asked about it said to install vvVeb in the root folder, but now I don't know which root (if you know what I mean). The public_html folder in the domain folder? Or the domain folder above that? 

-- If it's not practical to set it up to handle multiple sites, how do I at least set it up for one of them? 

-- Also, for those sites where I just want to use the editor (vvVeb.js, not the CMS), what folder should be the install folder? My results in trying to set it up are similar to setting up the CMS: It can't find the files listed in its own explorer.

-- Everything seems to install, but it's just not all available.

Sure would like to get this working. I sure like the program.

Thanks,
JS

**Comment by WebCrew** [comment]
I never had any issues with the installation, it is as easy as the setup of Joomla or WordPress and most other CMS. Oh an by the way the User Interface itself is so easy to understand there is as good as no learning curve.

Download the files from here: https://www.vvveb.com/
Now unzip and run the installer its that easy.

Kind regards and good luck, Andy :)

**Comment by givanz** [comment]
@WebCrew Thanks for your help.

> -- Where do I setup vvVeb? What I have tried gets the program to open, but all the pages "don't exist" even though they are listed in vvVeb. That includes the demo folder.

Do you refer to the Vvveb CMS or VvvebJs? Only VvvebJs has a demo folder.

> My results in trying to set it up are similar to setting up the CMS: It can't find the files listed in its own explorer.

VvvebJs needs to cloned/unzipped to the document root folder (htdocs) you probably tried to use a subfolder.

You also need to load editor.php instead of editor.html if you want to load previously saved files in file manager.

> The public site is in a sub-folder of the domain, called PUBLIC_HTML

Try to move the files directly to the subdomain folder, in some setups public_html is the root folder for main subdomain and for other subdomains the root folder is the subdomain folder itself without an additional public_html subfolder.

You can download the latest VvvebJs.zip release https://github.com/givanz/VvvebJs/releases/

> -- If it's not practical to set it up to handle multiple sites, how do I at least set it up for one of them?

VvvebJs is a page builder library meant to be extended or integrated into an existing CMS, it has only basic page saving functions provided as a base for further development, it does not have multi site support.

> -- I would like to set this up so that I can manage the CMS for multiple sites (domains) on the ISP. Again, the folder layout is unclear to me. The response I got when I asked about it said to install vvVeb in the root folder, but now I don't know which root (if you know what I mean). The public_html folder in the domain folder? Or the domain folder above that?

If you need more advanced features like multi site you can use Vvveb CMS.

In this case you don't need to install it for each subdomain, you only need to install for the main domain by unzipping to the main/top level public_html folder.

You can then add a new site for each  subdomain from `admin > settings > sites` after adding the subdomain in the dns/domain/cpanel settings.
